code/datums/components/listen_and_repeat.dm

```diff
@@ -2,12 +2,21 @@
 #define MAX_SPEECH_BUFFER_SIZE 500
 /// Tendency we have to ignore radio chatter
 #define RADIO_IGNORE_CHANCE 10
+/// The line we will re-iterate
+#define MESSAGE_LINE "line"
+/// the tts voice it should be said in
+#define MESSAGE_VOICE "voice"
+/// the tone it should be said in
+#define MESSAGE_PITCH "pitch"
+
+GLOBAL_LIST_INIT(invalid_voice, list(
+	MESSAGE_VOICE = "invalid",
+	MESSAGE_PITCH = 0,
+))
```

Comment:
Echoing san7890 here, this could work better as a static variable. Like, come on, this beaut of a PR has been open for a month now.